target_languages.json

```diff
@@ -0,0 +1,41 @@
+{
+  "target_languages": [
+    "Afrikaans",
+    "Arabic",
+    "Catalan",
+    "Chinese (simplified)",
+    "Croatian",
+    "Czech",
+    "Danish",
+    "Dutch",
+    "Esperanto",
+    "Estonian",
+    "Filipino",
+    "Finnish",
+    "French",
+    "German",
+    "Greek",
+    "Hindi",
+    "Hungarian",
+    "Icelandic",
+    "Italian",
+    "Japanese",
+    "Korean",
+    "Latvian",
+    "Lithuanian",
+    "Malay",
+    "Norwegian",
+    "Polish",
+    "Portugese",
+    "Romanian",
+    "Russian",
+    "Serbian",
+    "Slovak",
+    "Slovenian",
+    "Somali",
+    "Swedish",
+    "Turkish",
+    "Ukranian",
+    "Vietnamese"
```

Comment:
Where does the language list come from? I would appreciate a note in README (because json files can't contain comments).